src/material/radio/radio.html

```diff
@@ -5,7 +5,7 @@
   <div class="mat-radio-container">
     <div class="mat-radio-outer-circle"></div>
     <div class="mat-radio-inner-circle"></div>
-    <input #input class="mat-radio-input cdk-visually-hidden" type="radio"
+    <input #input class="mat-radio-input" type="radio"
```

Comment:
@jelbourn I don't remember why we ended up with `cdk-visually-hidden` in the first place, but from my testing, screen readers treat it exactly the same as when the input is `opacity: 0`. Given that we have the same issue in the other input-based components (slide toggle and checkbox), would it make sense to stop using it on native inputs?

Reply:
AFAIK we can't rely on just `opacity: 0`. I looked through my notes with Google a11y folks and had this from 2018:
```
Accessibility tree visibility of elements with `opacity: 0` across screen readers?
- Cannot rely on this.
- Possibly could set to .001
- Probably JAWS also, potentially in IE only
```

Which I interpret as JAWS treating `opacity: 0` as fully hidden.

Reply:
I know that IE used to have some security restrictions if an `input[type="file"]` has `opacity: 0`. I wonder whether that's not related to your notes.